ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -700,17 +700,6 @@ <h3 class="title">
                   >
                     <span>Convert to interleave</span>
                   </button>
-                  <button
-                    *ngIf="
-                      interleaveStatus.tableInterleaveStatus &&
-                      !interleaveStatus.tableInterleaveStatus.Possible
-                    "
-                    [disabled]="element.get('spName').value === ''"
-                    mat-menu-item
-                    (click)="convertToFk()"
-                  >
-                    <span>Convert to Foreign Key</span>
-                  </button>
                 </mat-menu>
```

Comment:
In case of two foreign keys, after interleaving one the second foreign key appears non adjacent to the source one: https://drive.google.com/file/d/19YkXKfZCM-BLLh_EKM-GXsK3z9astJ52/view?usp=share_link&resourcekey=0-oQLq3JDk0YtDSc8dSXz1Lg